src/utils/notifications.ts

```diff
@@ -72,6 +72,23 @@ export const handlePushManagerUnsubscribe = async (hide: () => void) => {
   }
 };
 
+export const blockNotification = async () => {
+  const data = {
+    accountId: 'erditkurteshi.near',
+    endpoint: '/preferences/set',
+    dapp: 'mention',
+    block: true,
+  };
```

Comment:
In general if my previous comment makes sense I'd propose this change:
```suggestion
export const manageNotification = async (accountId, endpoint, notificationType, block) => {
  const data = {
    accountId,
    endpoint,
    dapp: notificationType,
    block,
  };
```

Also it's good to add types to this params. And I think it's helpful to specify the endpoints in types